src/initialize.ts

```diff
@@ -43,10 +44,23 @@ export const initialize = async () => {
 const withState = <T, R>(f: (state: State, ...args: T[]) => R) =>
   (...args: T[]) => f(store.getState(), ...args)
 
+/** Set cursor function that is exposed to window for puppeteer testing. */
+const setCursorFirstMatch = (unrankedPath: string[]) => store.dispatch(setCursorFirstMatchActionCreator(unrankedPath))
+
+/** Imports text on given unranked path. Used in puppeteer testing. */
+const importTextFirstMatch = (unrankedPath: string[], text: string) => store.dispatch(importOnFirstMatchPathActionCreator(unrankedPath, text))
```

Comment:
Maybe rename `importTextFirstMatch` to `paste` to make the tests more readable and less implementation-specific?

Reply:
Sure.